<launch>

    <arg name="interact_venv" default="/home/$(env USER)/miniconda3/envs/asr/bin/python" doc="交互测试节点的Python环境路径"/>
    
    <!-- 依赖：interact (wakeup, stt, tts) -->
    <include file="$(find cerebellum_pkg)/launch/interact.launch"/>

    <!-- 主交互测试节点 -->
    <node name="interact_test_node" 
          pkg="cerebellum_pkg" 
          type="interact_test_node.py"
          launch-prefix="$(arg interact_venv)"
          output="screen"
          required="true">
      <param name="default_stt_wav_path" type="str" value="$(env YANBOT_WS)/last_heard_audios/stt.wav" />
    </node>

</launch>
